docs/en/01_Working_with_projects.md/10_Security.md

```diff
@@ -78,7 +78,7 @@ Uploaded files have their extension checked against known MIME types in the `HTT
 This basically means the file contents are checked to ensure the extension matches. For example, if you rename an image
 `test.jpg` to `test.txt` and attempt to upload it, the file will be rejected.
 
-Please see [technical docs for adding extensions](how-tos/adding_an_allowed_extension) for more information on
+Please see [technical docs for adding extensions](/how-tos/adding_an_allowed_extension) for more information on
```

Comment:
In one of the files further down you're using `how_tos` but `how-tos` here - assume it should be the former?

Reply:
I tested all urls in the browser and I recall that only the last part needs to be converted to underscores.